pom.xml

```diff
@@ -352,7 +352,7 @@
                 <plugin>
                     <groupId>com.github.eirslett</groupId>
                     <artifactId>frontend-maven-plugin</artifactId>
-                    <version>1.4.CODICE_1</version>
+                    <version>1.6.CODICE</version>
                     <executions>
                         <execution>
                             <id>install node and yarn</id>
```

Comment:
The configuration should be updated to allow over-ride of Yarn retrieval location. 

See DDF for reference: 
https://github.com/codice/ddf/blob/master/pom.xml#L514
VERSUS
https://github.com/codice/alliance/blob/cdaecef3ef0458f646a534785c3b4c899b8e75e8/pom.xml#L362

Reply:
done

Reply:
Thanks @bdeining!